### PR TITLE
Object Model Id's (Intellisense)

### PR DESCRIPTION
**Pull request by LukeSeers**
Bringing object models to your finger tips is amazing, but we have kinda 2 problems. What's your thoughts @NanoBob?

1. As we know GTA SA has a lot of objects, which in return will create a **HUGE** Enum file. 

2. I've also looked around to see if we could nest Enums, but sadly that's not a thing: [dotnet/roslyn/issues/13192](https://github.com/dotnet/roslyn/issues/13192).

If we was able to nest Enums it would not help in regards of a **HUGE** Enum file, but it would mean it's more readable and we could also categories models like: Beach and Sea, Buildings, Miscellaneous etc.

### CODE REVIEW

**Comment by NanoBob** [comment]
I'm not too sure if doing this would add additional value, most of the object names aren't that descriptive of what they are that I would need to see it in order to know which object it is anyway. Especially given how many world models there are in the game.

Although on the other hand there is no down side to adding the enum, but I think if we did we should also support a constructor with just the `ushort` (even though you could easily cast to it) since I think usage of it would be limited.

**Review by LukeSeers — COMMENTED** [review]
https://github.com/mta-slipe/Slipe-Server/pull/132/commits/217aaf6c093f8979b9a66f801659399d0f24a3e8 This contains all of the models from [wiki/Object_Ids](https://wiki.multitheftauto.com/wiki/Object_Ids).  
Since there is loads of id's in the Enum I made them collapsible in there respected categories.

File: SlipeServer.Console/ServerTestLogic.cs [hunk 1]
```diff
@@ -103,6 +103,9 @@ private void SetupTestElements()
             PedModel randomPedModel = (PedModel) new Random().Next(enumLength);
             new Ped(randomPedModel, new Vector3(10, 0, 3)).AssociateWith(server);
             
+            WorldModel model = new WorldModel(ObjectModel.Drugred);
+            new WorldObject(model.Id, new Vector3(15, 0, 3)).AssociateWith(server);
+
```
Comment (LukeSeers) [inline SlipeServer.Console/ServerTestLogic.cs:108]: @NanoBob Was this what you had in mind? I just want to double check if I've done the constructor correctly.

Comment (NanoBob) [reply SlipeServer.Console/ServerTestLogic.cs:108]: It is not, what I had in mind is just having a `WorldObject` constructor overload with a `ushort` as well, besides the one with the enum.

Comment (LukeSeers) [reply SlipeServer.Console/ServerTestLogic.cs:107]: Sorry I some how miss understood what you said. I hope that https://github.com/mta-slipe/Slipe-Server/pull/132/commits/813cba302f867f15f91417397d5d9ab0f2944101 https://github.com/mta-slipe/Slipe-Server/pull/132/commits/bba06ebf066f28b4d89b40a4d89bf223fa8d4957 is correct now.

**Pull request merged**